app/src/sharedTest/java/org/oppia/android/app/devoptions/MarkChaptersCompletedFragmentTest.kt

```diff
@@ -901,6 +904,70 @@ class MarkChaptersCompletedFragmentTest {
     }
   }
 
+  @Test
+  fun testMarkChaptersCompletedFragment_arguments_workingProperly() {
```

Comment:
Rename this and other tests to be more descriptive.
```suggestion
  fun testFragment_fragmentLoaded_verifyCorrectArgumentsPassed() {
```

Reply:
Ok

Reply:
Done.